include/mgos_freertos.h

```diff
@@ -23,8 +23,35 @@
 #include "queue.h"
 #include "task.h"
 
+#include "mongoose.h"
 #include "mgos_init.h"
 
+#ifndef MGOS_TASK_STACK_SIZE_BYTES
+#define MGOS_TASK_STACK_SIZE_BYTES 8192
+#endif
+
+#ifndef MGOS_TASK_PRIORITY
+#define MGOS_TASK_PRIORITY 5
+#endif
+
+#ifndef MGOS_TASK_QUEUE_LENGTH
+#define MGOS_TASK_QUEUE_LENGTH 32
+#endif
+
+#ifndef MGOS_MONGOOSE_MAX_POLL_SLEEP_MS
+#define MGOS_MONGOOSE_MAX_POLL_SLEEP_MS 1000
+#endif
+
+#ifndef MGOS_EARLY_WDT_TIMEOUT
+#define MGOS_EARLY_WDT_TIMEOUT 30 /* seconds */
+#endif
+
+#if CS_PLATFORM == CS_P_ESP32
```

Comment:
leave this in the source file, it not really tunable.

Reply:
Are you talking about the MGOS_EARLY_WDT_TIMEOUT or CS_PLATFORM ?

It's not meant to be tuned... I just want to be able to know the value...

Reply:
STACK_SIZE_UNIT. it's not really a tunable, it is hardcoded in freertos port and we just have to deal with it.

Reply:
CS_PLATFORM is needed to determine STACK_SIZE_UNIT, which i need.
But i don't really need MGOS_EARLY_WDT_TIMEOUT, but is there reason to conceal it?

Reply:
ah, you need STACK_SIZE_UNIT for your code... fair enough.
then let's keep it public, but please rename it to MGOS_TASK_STACK_SIZE_UNIT

Reply:
In my module i have code similar to this from `mgos_freertos.c` and it does need STACK_SIZE_UNIT see:

```C
xTaskCreateStatic(mgos_task, "mgos",
                   MGOS_TASK_STACK_SIZE_BYTES / STACK_SIZE_UNIT, NULL,
                    MGOS_TASK_PRIORITY, mgos_task_stack, &mgos_task_tcb);  
```

Reply:
> please rename it to MGOS_TASK_STACK_SIZE_UNIT

should have seen this coming :) i have pushed fixed version now.